etc/encoding/adaptive-streaming-movies.properties

```diff
@@ -108,33 +108,33 @@ profile.studio.adaptive-parallel.http.suffix.360p-quality = -360p-quality.mp4
 profile.studio.adaptive-parallel.http.jobload = 4
 profile.studio.adaptive-parallel.http.ffmpeg.command.if-height-geq-1800 = \
   -filter:v scale=w=3840:h=trunc(3840/dar/2)*2,setsar=1,fps=25 \
-    -c:v libx264 -preset slower -tune film -pix_fmt yuv420p \
-    -x264opts keyint=25:min-keyint=25:no-scenecut -crf 20 -maxrate 14800k -bufsize 18000k \
-    -c:a aac -b:a 196k -ac 2 \
+    -c:v libx264 preset faster -tune film -pix_fmt yuv420p \
+    -x264opts keyint=25:min-keyint=25:no-scenecut -crf 23 -maxrate 14800k -bufsize 29600k \
+    -c:a aac -b:a 128k -ac 2 \
```

Comment:
:+1:  for `128k` audio for speech. Since we are only recording a microphone which is unlikely to be a stereo signal, should we go for mono? And if we go for mono, should we go for `96k`? So many questions :D

Reply:
How much are we *really* saving by dropping to mono?  I'd leave it at stereo, but I'm happy to see the drop to 128.

Reply:
The idea was that mono will give you better quality for the same fixed bandwith. Especially if the content is really mono anyway.

Reply:
I agree with @gregorydlogan. Dropping to 128k make sense. I would prefer to keep stereo.